bazel_tools/client_server/runner_with_port_file/Main.hs

```diff
@@ -28,7 +28,7 @@ main = do
     let portFile = tempDir </> "portfile"
     let interpolatedServerArgs = map (replace "%PORT_FILE%" portFile) splitServerArgs
     let serverProc = proc serverExe interpolatedServerArgs
-    withCreateProcess serverProc $ \_stdin _stdout _stderr ph -> do
-      port <- readPortFile ph maxRetries portFile
+    withProcessTerm serverProc $ \ph -> do
```

Comment:
I'm confused. If you want to wait for it to finish, shouldn't you use `withProcessWait` here?

Reply:
I want to kill it gracefully by sending SIGTERM and then wait for it to finish. That’s what `withProcessTerm` does. `withProcessWait` on the other hand does not send SIGTERM and only waits for it to finish which for something like sandbox will never happen on its own.

Reply:
Ah, understood. Thanks for explaining!